### PR TITLE
Add safe-transmute project group

### PR DESCRIPTION
**Pull request by rylev**
Adding new project group in similar way as #193.

cc @nikomatsakis @joshtriplett

### CODE REVIEW

File: teams/wg-safe-transmute.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name = "safe-transmute"
```
Comment (nikomatsakis) [inline teams/wg-safe-transmute.md]: Huh, I thought that these had to be named wg-foo ...

Comment (pietroalbini) [reply teams/wg-safe-transmute.md]: It's supposed to...

Comment (rylev) [reply teams/wg-safe-transmute.md]: @pietroalbini @nikomatsakis whoops that's my fault. Strange that the CI passes....Should I change it?

Comment (nikomatsakis) [reply teams/wg-safe-transmute.md]: Yes, might as well. It is curious that CI passes though, I got errors when I ran `cargo run -- check` locally

**Comment by XAMPPRocky** [comment]
Thank you for your PR! One slight problem with this PR is that the file is using the `.md` extension instead of `.toml`. 😄

**Pull request merged**